client/config/config.go

```diff
@@ -18,6 +24,10 @@ func DefaultConfig() *Config {
 		Output:                "text",
 		Node:                  "tcp://localhost:26657",
 		BroadcastMode:         "sync",
+		GRPC: GRPCConfig{
+			Address:  "",
```

Comment:
It's weird imho to have this fallback to something if empty. Imho we should directly set the default value here

Reply:
The current default values for the gRPC client are these: https://github.com/cosmos/cosmos-sdk/blob/fde3134dadd2b538d20c32589719d073c06700ae/client/flags/flags.go#L108

Reply:
Ok, but what is the actual default value used?
It just reads weird when it is in a config file to have empty fields and fallback to something, unless we add a comment explaining what is the default when left empty (and if we do that, why not just set the default)

Reply:
You're right, it's confusing. Additionally, the zero values for string and boolean are what I set there, so I removed that redundant assignment